### ISSUE TITLE
[Accessibility]: Use of Alert component for non-alerting status 

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35259]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513

### Problem description

We currently use Alert components for their visual properties, but they have the side effect of grabbing attention from the screen-reader due to `role="alert"`. We should migrate away from the Alert component for non-alerting status indications in the upload and indexes list and details pages.

Examples of alerts to address:
Commit graph status
![image](https://user-images.githubusercontent.com/18282288/167697151-763e94bc-dbc7-493f-a34f-a500dd2cb1ba.png)

Upload/Index status
![image](https://user-images.githubusercontent.com/18282288/167697227-4b6e0412-45a1-4901-9b7a-42a96162a878.png)

Examples of alerts that make sense:
![image](https://user-images.githubusercontent.com/18282288/167697309-bea3f123-265e-499d-8d38-7b69c8856530.png)

### Expected behavior

When navigating to a page with these Alert components for non-alerting statuses, the screen-reader first reads the alert message, which doesn't make much sense. Instead, it should start as normal from the start of the page (or wherever it would start normally)

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@Strum355 This PR will fix: https://github.com/sourcegraph/sourcegraph/pull/35410
